README.md

```diff
@@ -9,15 +9,14 @@ This repository holds the source for this Implementation Guide, part of the fami
 
 ### What does it do?
 
--  TBC
-
+-   DNS-DS provides a simple and flexible method to discover NMOS RDS Services.
```

Comment:
Thanks Brad, I pulled in this change.  There is also a way on github to suggest changes directly to the branch and then the changes can be very easily updated on the branch.  I'm not sure how that works but maybe @peterbrightwell  knows? If not I can check with Nikita and the EDID groups since I know they are using that feature.